test/e2e/interface-language.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from '@playwright/test';
+import { test } from './utils/fixtures';
+import { LoginPage } from './pages/login.page';
+import { PageHeader } from './components/page-header.component';
+import { ForgotPasswordPage } from './pages/forgot-password.page';
+
+test.describe('Interface Language picker (LF only so far)', () => {
```

Comment:
Let's just call it "Interface language picker"

```suggestion
test.describe('Interface Language picker', () => {
```